documentation/src/main/wiki/UpgradeNotes.md

```diff
@@ -6,6 +6,9 @@ necessary configuration changes that were introduced in comparison to previously
 You may see all changes in the [Changelog](https://github.com/Cognifide/aet/blob/master/CHANGELOG.md).
 
 ## Unreleased
+Changes:
+* Counting line and column number of an accessibility issue occurrence in [[Accessibility Collector|AccessibilityCollector]] has been improved. 
+Columns are now indexed starting from 1. Bug with two identical issues on a single page yielding the same line and column number has been fixed.
```

Comment:
* Please provide some links here to related github issue(s) :)
* Pleasy explicitly explain here how users should change their suite.xml when upgrading from previous AET version - e.g. "When upgrading from previous AET version, you may need to decrease `column` value in <accessiblility-filter>s by 1 and change `line` value to match (...)"

Reply:
Done